bootstrap/bootstrap.go

```diff
@@ -946,9 +951,63 @@ func hasGitSubmodules(sh *shell.Shell) bool {
 	return fileExists(filepath.Join(sh.Getwd(), ".gitmodules"))
 }
 
+func (b *Bootstrap) referenceRepository() (string, error) {
+	path := os.Getenv(`BUILDKITE_REPO_MIRROR_PATH`)
```

Comment:
Is this different from `BUILDKITE_REPOS_PATH` set at `agent/job_runner.go#444`?

Reply:
This is the singular form, this is a full path to the reference repository.

Reply:
It's the fully realized path to the reference repository vs the dir that repositories are checked out to. Possibly badly named though.